core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -73,6 +73,7 @@ public PartitionData copy() {
   private long[] splitOffsets = null;
   private int[] equalityIds = null;
   private byte[] keyMetadata = null;
+  private Integer sortOrderId;
```

Comment:
I think this can be an int because we have a default that is always valid, `0`.

Reply:
Same outcome as the comment below. The problem is with position delete files. Nevermind.